.lando.yml

```diff
@@ -6,12 +6,15 @@ config:
   via: nginx
   webroot: web
   database: "mariadb:10.3"
-  composer_version: 2
   xdebug: off
   config:
     php: .lando/php.ini
 
 tooling:
+  composer:
+    description: "Runs Composer commands"
+    cmd:
+      - appserver: ./vendor/bin/composer
```

Comment:
This doesn't work if I need to remove vendor (every time composer hangs)
```
rm -rf vendor
❯ lando composer install
OCI runtime exec failed: exec failed: unable to start container process: exec: "./vendor/bin/composer": stat ./vendor/bin/composer: no such file or directory: unknown
```

Reply:
Looks like @tparssinen has proper fix in place here: https://github.com/wunderio/drupal-project/pull/329/files#diff-307b4548b91e997f23c35d90ed5e9a044e156b90425c9923d6b85d7dc11e67e2R22

Reply:
Committing update soon.